### ISSUE TITLE
Download blocked

### ISSUE THREAD

**Issue by kwheelan** [closup/process-xbrl#139]
After uploading and converting the Excel files, the app does not allow the user to download the XBRL when clicking "Download File" (at least in Chrome). I think this is an HTML issue. 

In Chrome, clicking download does nothing. The JS console logs this error:
```
complete:1 Mixed Content: The site at 'https://xbrl-converter-22f50351e04c.herokuapp.com/' was loaded over a secure 
connection, but the file at 'https://xbrl-converter-22f50351e04c.herokuapp.com/static/sessions_data/0fe00400-a65f-4fb4-887b-
dc2ccc210587/output/converted_xbrl.zip' was redirected through an insecure connection. This file should be served over HTTPS. 
See https://blog.chromium.org/2020/02/protecting-users-from-insecure.html for more details.
```

In Safari, it allows the download with a warning:

<img width="1257" alt="Screenshot 2024-11-26 at 10 48 09 AM" src="https://github.com/user-attachments/assets/c0a713fa-ed0e-4b8f-b184-619c178e7c46">

The download itself is the correct file.

**Comment by kwheelan** [comment]
@lucakato Chrome doesn't block the template download, so I bet the output is blocked because it's a zip. Just noting in case it's helpful for future troubleshooting

**Comment by lucakato** [comment]
@kwheelan  Could you test branch #139 to see if download is blocked for you there? Changed to using send-file.

```
        return send_file(
            zip_path,
            mimetype='application/zip',
            as_attachment=True,
            download_name='converted_xbrl.zip'
        )
```

**Comment by kwheelan** [comment]
@lucakato Yes! Fixed it for me! I'll merge into `dev` to test the deployment there.
